### PR TITLE
Add option to terminate app execution on svcBreak

### PR DESCRIPTION
**Pull request by kynex7510**
This commit adds a checkbox in the debugging configuration screen for terminating the app when `svcBreak` is executed, instead of just logging the issue and praying everything goes well (this is retained as the default behaviour). It's especially useful when debugging faulty homebrew, so logs don't become excessively long.

- - -
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/6333)

### CODE REVIEW

**Comment by GPUCode** [comment]
Please add a screenshot of the UI changes

**Comment by kynex7510** [comment]
My bad: 
image

**Comment by SleepingSnakezzz** [comment]
Please also add a warning in () telling users **not** to enable this if they don't know what it does. Since we direct users there for log files, I can already tell that this will create problems.

**Comment by kynex7510** [comment]
Is this ok?
image

**Comment by SachinVin** [comment]
Does this need to be an option? If it is not something that happens during normal execution on commercial games, we can probably just do something similar to the "Fatal Error" dialog https://github.com/citra-emu/citra/blob/master/src/core/hle/service/err_f.cpp#L172

**Comment by kynex7510** [comment]
One hypothesis I've been told is that it may be ignored by default to overcome emulation bugs.

File: src/core/hle/kernel/svc.cpp [hunk 1]
```diff
@@ -1134,6 +1135,9 @@ void SVC::Break(u8 break_reason) {
         break;
     }
     LOG_CRITICAL(Debug_Emulated, "Break reason: {}", reason_str);
+
+    if (Settings::values.terminate_on_svcbreak.GetValue())
+        system.RequestShutdown();
```
Comment (B3n30) [inline src/core/hle/kernel/svc.cpp:1140]: nit: maybe log here so when looking at the end of the log one immediately knows why it emulation stopped

**Comment by FearlessTobi** [comment]
Superseded by https://github.com/citra-emu/citra/pull/6364.
Thanks a lot for your contribution though!

**Pull request closed without merging**